moonlight-common/moonlight-common_mac.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,510 @@
+// !$*UTF8*$!
```

Comment:
Is there a way to avoid having a duplicate moonlight-common project for Mac?

Reply:
About this one i really don't know exactly, i tried to use the same one but the linker complained about the architecture, maybe there is a way, but i don't know. The answer to this would possibly also help with the duplicate opus lib.